app/code/Magento/CatalogInventory/Api/Data/InventoryInterface.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Magento\CatalogInventory\Api\Data;
+
+use Magento\CatalogInventory\Api\LocationInformationInterface;
+use Magento\CatalogInventory\Api\ConfigurationInterface;
+
+/**
+ * Inventory interface
+ *
+ * @api
+ */
+interface InventoryInterface extends LocationInformationInterface
```

Comment:
Please, calrify: how InventoryInterface and WarehouseInterface are connected (or they are not supposed to be)?